docs/docs/constraints.md

```diff
@@ -28,108 +28,108 @@ Regex is allowed for LIKE and UNLIKE operators; to match ANY value, use the stri
+You can also use this attribute to tie an application to a specific node by using the hostname property:
 
 ``` bash
 $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
     "id": "sleep-cluster",
     "cmd": "sleep 60",
     "instances": 3,
-    "constraints": [["rack_id", "CLUSTER", "rack-1"]]
+    "constraints": [["hostname", "CLUSTER", "a.specific.node.com"]]
   }'
 ```
 
-You can also use this attribute to tie an application to a specific node by using the hostname property:
+### LIKE operator
+**Value** (required): A regular expression for the value of the attribute.
+
+`LIKE` accepts a regular expression as parameter and allows you to run your tasks only on the agent nodes whose field values match the regular expression.
 
 ``` bash
 $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
-    "id": "sleep-cluster",
+    "id": "sleep-group-by",
     "cmd": "sleep 60",
     "instances": 3,
-    "constraints": [["hostname", "CLUSTER", "a.specific.node.com"]]
+    "constraints": [["rack_id", "LIKE", "rack-[1-3]"]]
   }'
 ```
 
-### GROUP_BY operator
+### UNLIKE operator
+**Value** (required): A regular expression for the value of the attribute.
 
-`GROUP_BY` can be used to distribute tasks evenly across racks or datacenters for high availability:
+Just like `LIKE` operator, but only run tasks on agent nodes whose field values don't match the regular expression.
 
 ``` bash
 $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
     "id": "sleep-group-by",
     "cmd": "sleep 60",
     "instances": 3,
-    "constraints": [["rack_id", "GROUP_BY"]]
+    "constraints": [["rack_id", "UNLIKE", "rack-[7-9]"]]
   }'
 ```
 
-Marathon only knows about different values of the attribute (e.g. "`rack_id`") by analyzing incoming offers from Mesos. If tasks are not already spread across all possible values, specify the number of values in constraints. If you don't specify the number of values, you might find that the tasks are only distributed in one value, even though you are using the GROUP_BY constraint. For example, if you are spreading across 3 racks, use:
+### MAX_PER operator
+**Value** (required): An integer, for example `"2"`.
+
+`MAX_PER` accepts a number as parameter which specifies the maximum size of each group. It can be used to limit tasks across racks or datacenters.
 
 ``` bash
 $ curl -X POST -H "Content-type: application/json" localhost:8080/v2/apps -d '{
     "id": "sleep-group-by",
     "cmd": "sleep 60",
     "instances": 3,
-    "constraints": [["rack_id", "GROUP_BY", "3"]]
+    "constraints": [["rack_id", "MAX_PER", "2"]]
   }'
 ```
 
+### UNIQUE operator
+**Value** (optional): A string that defines the value of the attribute. This attribute must be unique. If you specify `hostname:UNIQUE`, any value you define will be ignored.
```

Comment:
Marathon actually throws an error when a value is provided. So the value is _not_ optional. It is required to not be applied!